### ISSUE TITLE
options.uri is a required argument

### ISSUE THREAD

**Issue by bdefore** [browserstack/browserstack-cypress-cli#68]
I'm unable to get started with Browserstack Cypress. I'm running 1.5.0. I've tried using browerstack-cypress-cli both locally and globally, installed via yarn. Here's my config (mostly unchanged from the docs):
```
{
  "auth": {
    "username": "my_username_here",
    "access_key": "my_key_here"
  },
  "browsers": [{
      "browser": "chrome",
      "os": "Windows 10",
      "versions": ["latest", "latest-1"]
    },
    {
      "browser": "firefox",
      "os": "OS X Mojave",
      "versions": ["latest", "latest-1"]
    },
    {
      "browser": "edge",
      "os": "OS X Catalina",
      "versions": ["latest", "latest-1"]
    }
  ],
  "run_settings": {
    "cypress_config_file": "./src/path/to/monorepo/project/cypress.json",
    "project_name": "Project",
    "build_name": "Build",
    "parallels": "5"
  }
}
```
And the output:
```
→ ./node_modules/.bin/browserstack-cypress run
No env file present for the current environment:  development 
 Falling back to .env config
No env file present for the current environment:  development
[10/9/2020, 2:32:42 PM] - info: Reading config from /Users/▉▉/▉▉/browserstack.json
[10/9/2020, 2:32:42 PM] - info: browserstack.json file is validated
[10/9/2020, 2:32:43 PM] - error:  
{
  message: 'options.uri is a required argument',
  stack: 'Error: options.uri is a required argument\n' +
    '    at Request.init (/Users/▉▉/▉▉/node_modules/request/request.js:231:31)\n' +
    '    at new Request (/Users/▉▉/▉▉/node_modules/request/request.js:127:8)\n' +
    '    at request (/Users/▉▉/▉▉/node_modules/request/index.js:53:10)\n' +
    '    at Function.post (/Users/▉▉/▉▉/node_modules/request/index.js:61:12)\n' +
    '    at /Users/▉▉/▉▉/node_modules/browserstack-cypress-cli/bin/helpers/zipUpload.js:28:13\n' +
    '    at new Promise (<anonymous>)\n' +
    '    at Object.uploadCypressZip [as zipUpload] (/Users/▉▉/▉▉/node_modules/browserstack-cypress-cli/bin/helpers/zipUpload.js:10:10)\n' +
    '    at /Users/▉▉/▉▉/node_modules/browserstack-cypress-cli/bin/commands/runs.js:55:28\n' +
    '    at runMicrotasks (<anonymous>)\n' +
    '    at processTicksAndRejections (internal/process/task_queues.js:97:5)'
}
[10/9/2020, 2:32:43 PM] - error: Zip Upload failed.
[10/9/2020, 2:32:44 PM] - info: Zip file deleted successfully.
```
This is pointing to a Cypress config I'm able to run via Cypress directly outside of Browserstack.

**Comment by suryart** [comment]
Are you using/setting ` NODE_ENV` in the shell? It seems like the custom env is picking up the NODE_ENV and trying to do a lookup on a URL that doesn't exist.

If yes, then can you try without setting it? Is there any specific case for using it?

**Comment by suryart** [comment]
@bdefore - This issue should be fixed in [v1.5.1](https://www.npmjs.com/package/browserstack-cypress-cli/v/1.5.1). Let us know in case of any issues.

**Comment by bdefore** [comment]
thanks @suryart i can confirm i no longer get this issue.
